packages/trader/src/App/Components/Elements/market-countdown-timer.tsx

```diff
@@ -151,7 +152,9 @@ const MarketCountdownTimer = observer(
         let opening_time_banner = null;
         if (opening_time) {
             const formatted_opening_time = convertTimeFormat(opening_time);
-            const target_date = moment(new Date()).add(days_offset, 'days');
+            const target_date = moment(new Date())
+                .locale(getlangFromUrl()?.toLowerCase() ?? 'en')
```

Comment:
I was thinking about setting the moment locale globally. Are there any issues with that?

Reply:
Actually I was afraid a bit because it will change (translate) dates in calendar, contract audit and many-many  places outside of our team scope. Should I do this?